packages/main/src/plugin/api/provider-info.ts

```diff
@@ -50,6 +50,7 @@ export interface ProviderKubernetesConnectionInfo {
 export interface ProviderInfo {
   internalId: string;
   id: string;
+  extensionId: string;
```

Comment:
probably missing then the readonly attribute there

Reply:
done 👍